jcli/src/jcli_app/certificate/new_stake_pool_registration.rs

```diff
@@ -30,6 +30,14 @@ pub struct StakePoolRegistration {
         required = true
     )]
     pub owners: Vec<PublicKey<Ed25519>>,
+    /// public key of the operators(s)
+    #[structopt(
+        long = "operators",
```

Comment:
Not that important, but if possible can we keep also this in *singular* like we have it for *owner*.

Reply:
good catch